README.md

```diff
@@ -6,12 +6,41 @@ Kibosh acts as a pass-through layer on top of existing filesystems.  Usually,
 it simply forwards each request on to the underlying filesystems, but it can be
 configured to inject arbitrary faults.
 
-# Build Requirements
+# Environment
 
 Kibosh runs on Linux.  We do not currently support Mac or Windows.
 
+If running in a container environment, the container needs to be run with 
+additional options giving Kibosh access to FUSE.
+
+    --cap-add SYS_ADMIN     # this allows Kibosh to mount a FUSE filesystem
+    --device /dev/fuse      # this exposes Kibosh to the FUSE device
+    
+Alternatively, the container can be run with --privileged flag which grants
+Kibosh all the permissions it needs.
+
+    $ docker run --privileged -ti ubuntu /bin/bash
+
+# Build Requirements
+
 In order to build Kibosh, you must have the CMake build system installed, and a
-C compiler.  We also depend on the development libraries for FUSE.
+C compiler.  We also depend on the development libraries for FUSE.  Pkg-config is
+used by configure script to gather paths of required libraries.
+
+Dependencies can be installed with the following commands.  Run configure before
```

Comment:
Probably better to shorten this as "in Ubuntu, run `apt-get cmake libfuse-dev fuse pkg-config` to install the dependencies

Reply:
Edited as suggested.